web/tests/common.test.js

```diff
@@ -153,7 +153,6 @@ run_test("adjust_mac_tooltip_keys mac expected", ({override}) => {
         [["PgUp"], ["Fn", "↑"]],
         [["PgDn"], ["Fn", "↓"]],
         [["Ctrl"], ["⌘"]],
-        [["Alt"], ["⌘"]],
```

Comment:
I've removed the Alt mapping here and in the test below since there is actually no tooltip yet for shortcuts involving Alt.